docs/INSTALL.rst

```diff
@@ -99,27 +104,79 @@ setups, see ``docs/network-drivers.md``.
 Logging level and directory can be set in the ``[general]`` section. For more
 information view ``docs/logging.md``.
 
-The file should be placed at ``/etc/haas.cfg``; The ``haas.wsgi``
-script, described below, requires this. Awkwardly, the ``haas``
-command line tool expects the file to be present in its current
-working directory. This will be fixed in the next release, but for
-now, put the file in ``/etc`` and create a symlink to it in the
-HaaS user's home directory::
 
-  sudo ln -s /etc/haas.cfg /var/lib/haas/
+``haas.cfg`` file contains sensitive administrative information and should not be exposed to clients or 
+end users. Therefore, after placing the file at ``/etc/haas.cfg`` its 
+permissions should be set to read-only and ownership set to ``${HAAS_USER}``
+From source directory of haas as user root do the following::
+
+    (from /root/haas)
+    cp examples/haas.cfg /etc/haas.cfg
+    chown ${HAAS_USER}:${HAAS_USER} haas.cfg
+    chmod 400 haas.cfg
+    (run following command as ${HAAS_USER} from ${HAAS_HOME_DIR}
+    su - ${HAAS_USER}
+    ln -s /etc/haas.cfg .
+
+Authentication and Authorization
+--------------------------------
+
+HaaS includes a pluggable architecture for authentication and authorization.
+HaaS ships with two authentication backends. One uses HTTP basic auth, with
+usernames and passwords stored in the haas database. The other is a "null"
+backend, which does no authentication or authorization checks. This can be
+useful for testing and experimentation but *should not* be used in production.
+You must enable exactly one auth backend.
+
+In productions system where non-null backend is active, end users will have to include
+a username and password as additional parameters in ``client_env`` file to be able to 
+communicate with the haas server. This is user/password should be registered with the 
+haas auth backend using haas.
+
+
+Database Backend
+^^^^^^^^^^^^^^^^
+
+To enable the database backend, make sure the **[extensions]** section of
+``haas.cfg`` contains::
+
+  haas.ext.auth.database =
+
+Null Backend
+^^^^^^^^^^^^
+
+To enable the null backend, make sure **[extensions]** contains::
+
+  haas.ext.auth.null =
+
+Setting Up HaaS Database
+------------------------
+
+The only DBMS currently supported for production use is PostgreSQL. (SQLite is
+supported for development purposes *only*).
```

Comment:
I do not see why these two instances of supported need to be removed. If I remove what should I replace it with. without the words the sentence does not make sense. @henn help ?